Userland/Libraries/LibDeviceTree/DeviceTree.h

```diff
@@ -125,6 +132,33 @@ class DeviceTreeNodeView {
         return false;
     }
 
+    FlatPtr to_root_address(FlatPtr value) const;
+
+    Optional<DeviceTreeProperty> raw_reg() const { return get_property("reg"sv); }
+
+    Vector<Reg, 2> reg() const
+    {
+        auto reg_property = raw_reg();
+        if (!reg_property.has_value())
+            return {};
+
+        auto address_cells = get_property("#address-cells"sv);
+        auto size_cells = get_property("#size-cells"sv);
```

Comment:
The `reg` property uses the `#address-cells` and `#size-cells` of the parent node.
The dt spec also says this if these properties are missing:
> If missing, a client program should assume a default value of 2 for #address-cells, and a value of 1 for #size-cells.